src/main/scala/tilelink/Bundles.scala

```diff
@@ -37,15 +37,15 @@ object TLMessages
   def GrantData      = UInt(5) //                    .         => GrantAck
   def ReleaseAck     = UInt(6) //                    .
   def GrantAck       = UInt(0) //                         .
- 
+
```

Comment:
Please do not include unnecessary changes into a PR. It lead to spurious conflicts.